sync/v1.proto

```diff
@@ -863,6 +875,36 @@ message Rule {
   ];
 }
 
+message FileAccessWatchItem {
+  enum RuleType {
+    RULE_TYPE_UNSPECIFIED = 0;
+    RULE_TYPE_PATHS_WITH_ALLOWED_PROCESSES = 1;
+    RULE_TYPE_PATHS_WITH_DENIED_PROCESSES = 2;
+    RULE_TYPE_PROCESSES_WITH_ALLOWED_PATHS = 3;
+    RULE_TYPE_PROCESSES_WITH_DENIED_PATHS = 4;
+  }
+  message Process {
+    oneof identifier {
```

Comment:
This can't be a `oneof`. The attributes specified for a given process block are ANDed together

Reply:
> This would preclude writing policies that are more restrictive of the binary. A policy for the binary with the signing ID platform:com.apple.yes as long as its in /usr/bin/yes would be inexpressible.

Correct, this is the one use case the is precluded by keeping this a `oneof`.

The question is - do we think this is a good use case that people want/need? We haven't yet seen it used in the wild by anyone with configs that have been shared with us or discussed publicly. What would be the reason to allow a signed binary from only a specific location?

Reply:
We're ok with this limitation.